### DISCUSSION TITLE
Not able to connect to remote server

### DISCUSSION THREAD

**Discussion by labomatik** [spatie/ray#510] [Q&A]
I've installed Ray and successfully connected to a remote server but i don't have any data coming from the server.

Do you have any idea what to check?

My config:
Ray Version 1.14.7 (1.14.7)
Server 
![image](https://user-images.githubusercontent.com/179987/125327638-9e1dd000-e343-11eb-8f84-5104144854e8.png)

**Comment by AdrianMrn** [comment]
@AlexVanderbist Any idea?
